packages/react-core/src/components/AlertGroup/AlertGroup.tsx

```diff
@@ -10,6 +10,8 @@ export interface AlertGroupProps extends Omit<React.HTMLProps<HTMLUListElement>,
   children?: React.ReactNode;
   /** Toast notifications are positioned at the top right corner of the viewport */
   isToast?: boolean;
+  /** Dyanmic alerts need to be rendered in pre-existing live region */
```

Comment:
Instead of documenting how another component is related to this option, let's consider just explaining what this option does for this component. For example "isDynamic AlertGroup's turn the container into a live region so that changes to content within the AlertGroup, such as appending an Alert, are reliably announced to assistive technology".